nemoguardrails/rails/llm/config.py

```diff
@@ -1203,3 +1210,19 @@ def _generate_rails_flows(flows):
         flow_definitions.insert(1, _LIBRARY_IMPORT + _NEWLINE * 2)
 
     return flow_definitions
+
+
+def _is_file_ignored_by_railsignore(filename: str) -> bool:
+    # Default no skip
+    should_skip_file = False
+
+    # Load candidate patterns from railsignore
+    candidate_patterns = get_railsignore_patterns()
+
+    # Ignore colang, kb, python modules if specified in valid railsignore glob format
+    if filename.endswith(".py") or filename.endswith(".co") or filename.endswith(".kb"):
```

Comment:
Let's relax this extension check and only stick to the .railsignore

Reply:
done